src/components/Application.js

```diff
@@ -0,0 +1,330 @@
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      applicantName: '',
+      resume: '',
+      classYear: '',
+      major: '',
+      taken112: '',
+      taken330: '',
+      q1: '',
+      q2: '',
+      timeslots: {
+        saturday: {
+          "9 AM - 10 AM": false,
+          "10 AM - 11 AM": false,
+          "11 AM - 12 PM": false,
+          "12 PM - 1 PM": false,
+          "1 PM - 2 PM": false,
+          "2 PM - 3 PM": false,
+          "3 PM - 4 PM": false,
+          "4 PM - 5 PM": false,
+          "5 PM - 6 PM": false
+        },
+        sunday: {
+          "9 AM - 10 AM": false,
+          "10 AM - 11 AM": false,
+          "11 AM - 12 PM": false,
+          "12 PM - 1 PM": false,
+          "1 PM - 2 PM": false,
+          "2 PM - 3 PM": false,
+          "3 PM - 4 PM": false,
+          "4 PM - 5 PM": false,
+          "5 PM - 6 PM": false
+        }
+      },
+      validated: false,
+      error: null,
+      submitted: false
+    };
+  }
+
+  onChange = event => this.setState({ [event.target.name]: event.target.value });
+
+  onCheckboxChange = event => {
+    const { timeslots } = this.state;
+    const checked = timeslots[event.target.name][event.target.value];
+    timeslots[event.target.name][event.target.value] = !checked;
+    this.setState({ timeslots });
+  }
+
+  onSubmit = event => {
+    event.preventDefault();
+    const form = event.currentTarget;
+    if (form.checkValidity() === false) {
+      event.stopPropagation();
+    } else {
+      axios.post('/api/saveApplication', {
+        email: this.state.email,
+        name: this.state.applicantName,
+        resume: this.state.resume,
+        classYear: this.state.classYear,
+        major: this.state.major,
+        taken112: this.state.taken112,
+        taken330: this.state.taken330,
+        q1: this.state.q1,
+        q2: this.state.q2,
+        timeslots: this.state.timeslots
+      })
+      .then(res => console.log(res))
+      .catch(error => console.error(error));
+
+      this.setState({ submitted: true });
+      console.log(this.state);
+    }
+
+    this.setState({ validated: true });
+  };
+
+  render() {
+    const {
+      email,
+      applicantName,
+      classYear,
+      major,
+      taken112,
+      taken330,
+      resume,
+      q1,
+      q2,
+      timeslots,
+      validated,
+      submitted,
+      error
+    } = this.state;
+
+    const successMessage = (
+      <Container className="application-form-wrapper">
+        <Row>
+          <Col className="col-md-12 text-center">
+            <img src={logo} alt="UPE Logo" height="256" width="256" />
+            <h1>Application Submitted!</h1>
+          </Col>
+        </Row>
+        <Row className="thanks-text">
+          <Col className="col-md-12 text-center">
+            <p>Thank you for applying to join UPE, we will be getting back to you by Friday the XX with a timeslot for your interview. </p>
```

Comment:
Nit: remove comma splice
```suggestion
            <p>Thank you for applying to join UPE. We will be getting back to you by Friday the XX with a timeslot for your interview. </p>
```